vsix/ProjectTemplates/VC/Windows Universal/BlankApp/App.h

```diff
@@ -6,7 +6,8 @@ namespace winrt::$safeprojectname$::implementation
     struct App : AppT<App>
     {
         App();
-
+        void InitializeComponent();
+        void initialize_instance(){ InitializeComponent(); }
```

Comment:
This looks really hacky. Can't we make this a little more polished? Just specialize this for Xaml-derived classes that actually have a InitializeComponent method?

Reply:
We could remove this altogether for App, since it has an empty InitializeComponent.  My concern was with introducing any kind of breaking change.  The Xaml compiler may choose to provide an implementation at some point.  But yes, could remove the generality and just specialize for InitializeComponent.